app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -141,7 +134,11 @@ function DocumentCapture({ isAsyncForm = false, onStepChange = () => {} }: Docum
                     captureHints: submissionError.hints,
                     pii: submissionError.pii,
                   })(ReviewIssuesStep)
-                : ReviewIssuesStep,
+                : withProps({
```

Comment:
Surprised we needed to make this change. What was the issue with how it was before?

Reply:
tsc was throwing errors about wanting the extra properties here after I removed the background upload URL and key; I didn't dig into why.

Reply:
Done